app-frontend/src/app/components/vectors/vectorImportModal/vectorImportModal.html

```diff
@@ -0,0 +1,145 @@
+<div class="modal-scrollable-body">
+  <div class="modal-header">
+    <button type="button" class="close" aria-label="Close" ng-click="$ctrl.handleClose()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <h4 class="modal-title" ng-if="!$ctrl.resolve.project">
+      Upload Shape
+    </h4>
+  </div>
+
+  <!-- Body for SELECT_FILE -->
+  <div class="modal-body" ng-if="$ctrl.currentStepIs('SELECT_FILE')">
+    <div class="content">
+      <h3 class="no-margin">Select a zipped shapefile</h3>
+      <ng-container ng-if="!$ctrl.selectedFile">
+        <p>Select shape files from your computer. {{$ctrl.BUILDCONFIG.APP_NAME}} currently allows for uploading zipped shape files only.</p>
+        <br/>
+        <br/>
+
+        <div class="text-center">
+          <button class="btn btn-primary"
+                  ngf-multiple=false
+                  ng-model="$ctrl.selectedFile"
+                  ngf-pattern="'*.zip'"
+                  ngf-change="$ctrl.fileSelected($file)"
+                  ngf-select
+          >
+            Select File
+          </button>
+
+          <br/>
+          <br/>
+
+          or drag and drop file
+        </div>
+      </ng-container>
+      <ng-container ng-if="$ctrl.selectedFile">
+        <div class="list-group">
+          <div class="list-group-item">
+            <div class="list-group-overflow">
+              <input type="text"
+                     class="form-control"
+                     ng-attr-placeholder="Save as: {{$ctrl.selectedFile.name}}"
```

Comment:
This input doesn't get cleared if the chosen file is removed and a different file is chosen. I found this to be unexpected.